tcms/testplans/templates/testplans/get.html

```diff
@@ -153,6 +153,10 @@ <h2 class="card-pf-title">
                             <span class="fa fa-search"></span>
                             <span class="js-test-case-tester"></span>
                         </div>
+                        <div title="{% trans 'Automated' %}" class="list-view-pf-additional-info-item">
```

Comment:
title will always appear as "Automated" regardless of actual value. I can see how the original template is a bit misleading in this part.
Here you want both states `Manual` & `Automated` as data attributes (so they can get translated when the HTML is returned from the BE) but assign the appropriate value to the title attribute at runtime.